storage/google/cloud/storage/blob.py

```diff
@@ -941,14 +965,18 @@ def upload_from_file(self, file_obj, rewind=False, size=None,
             warnings.warn(_NUM_RETRIES_MESSAGE, DeprecationWarning)
 
         _maybe_rewind(file_obj, rewind=rewind)
+        predefined_acl = ACL.validate_predefined(predefined_acl)
```

Comment:
Hmm, I guess that does the Right Thing(TM) when the value passed in is `None`?

Reply:
Seemed like it and shown with the test.  it was code nicely written which I moved.